src/meili/map_matcher.cc

```diff
@@ -277,31 +277,43 @@ MapMatcher::OfflineMatch(
     const std::vector<Measurement>& measurements, uint32_t k) {
   mapmatching_.Clear();
 
-  const auto begin = measurements.begin(),
-               end = measurements.end();
-
-  if (begin == end) {
+  if (measurements.size() < 2)
```

Comment:
sorry about that! it seemed that a gps trace of 1 point wasnt very useful (since we do actually do map matching) but i can see why answering more requests is useful. sorry again